### PR TITLE
Feature/composer require command

### PR DESCRIPTION
**Pull request by nstapelbroek**
Hey Sven, 

This PR should fix the requested composer require command as issued in #4. 
### What has been done:
- Added a value-object to help me format the data
- Added an additional attachment to the hook, containing a `composer require` command.

selection_006

Let me know if you have any questions or comments.

### CODE REVIEW

File: app/Http/Controllers/SlackController.php [hunk 1]
```diff
@@ -83,30 +84,36 @@ public function auth(Request $request)
...
                 [
-                    'fallback' => $package['description'],
-                    'title' => $package['name'],
-                    'title_link' => $package['url'],
-                    'text' => $package['description'],
+                    'fallback' => $package->getDescription(),
+                    'title' => $package->getName(),
+                    'title_link' => $package->getUrl(),
+                    'text' => $package->getDescription(),
+                    "mrkdwn_in" => ["fields"],
```
Comment (svenluijten) [inline app/Http/Controllers/SlackController.php]: Could you switch these out to single quotes?

File: app/ValueObjects/Package.php [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            throw new \InvalidArgumentException('Missing package details in response');
+        }
+
+        $package = $packageDetails['package'];
+        $package['url'] = 'https://packagist.org/packages/' . $package['name'];
+        $package['downloads'] = $package['downloads']['total'];
+
+        return new self($package);
+    }
+}
```
Comment (svenluijten) [inline app/ValueObjects/Package.php]: please add a newline to the end of this file, I try to adhere to PSR-2 as much as possible 😄

**Comment by nstapelbroek** [comment]
Good catch, I've updated the PR :+1:

**Comment by svenluijten** [comment]
Thanks! Looks good to me :thumbsup:

**Pull request merged**